src/drawing_tool/graphic/eraser.cpp

```diff
@@ -17,7 +17,7 @@ namespace Graphic {
 Eraser::Eraser(GraphicMap::VisibleGraphicLayer& visibleGraphicLayer)
     : Graphic::GraphicTool(QIcon(":/icons/icon_eraser.png"),
                            visibleGraphicLayer)
-{}
+{m_mouseClicked = false;}
```

Comment:
Yes, that works, but it's not in the initialization list!

I actually expected this:

```cpp
Eraser::Eraser(GraphicMap::VisibleGraphicLayer& visibleGraphicLayer)
    : Graphic::GraphicTool(QIcon(":/icons/icon_eraser.png"),
                           visibleGraphicLayer), m_mouseClicked(false)
```

You can have more information here: https://openclassrooms.com/fr/courses/1894236-programmez-avec-le-langage-c/1897606-creez-les-classes-partie-2-2#/id/r-1907275